rclcpp/src/rclcpp/timer.cpp

```diff
@@ -62,16 +81,16 @@ std::chrono::nanoseconds
 TimerBase::time_until_trigger()
 {
   int64_t time_until_next_call = 0;
-  if (rcl_timer_get_time_until_next_call(&timer_handle_, &time_until_next_call) != RCL_RET_OK) {
+  if (rcl_timer_get_time_until_next_call(timer_handle_.get(), &time_until_next_call) != RCL_RET_OK) {
     throw std::runtime_error(
-            std::string("Timer could not get time until next call: ") +
-            rcl_get_error_string_safe());
+      std::string("Timer could not get time until next call: ") +
+      rcl_get_error_string_safe());
```

Comment:
Did you run `ament_uncrustify` or the test suite on this? This is a known oddity of how uncrustify makes use format the code. I would actually expect it to fail the tests with this diff.